test/unit/analyze/test_concept_map.py

```diff
@@ -47,25 +57,13 @@ def test_concept_map_init():
     }
 
 
-@mock.patch("fhirpipe.analyze.concept_map.fhirpipe.global_config", mock_config)
-@mock.patch("fhirpipe.analyze.concept_map.requests.get", mock_api_get_maps)
-def test_fetch_concept_map(fhir_concept_map_gender):
-    concept_map = ConceptMap.fetch(fhir_concept_map_gender["id"])
-
-    assert concept_map == ConceptMap(fhir_concept_map_gender)
-
-    # should raise if not found
-    with pytest.raises(Exception, match="Error while fetching concept map nope: not found."):
-        ConceptMap.fetch("nope")
-
-
 @mock.patch("fhirpipe.analyze.concept_map.fhirpipe.global_config", mock_config)
 @mock.patch("fhirpipe.analyze.concept_map.requests.get", mock_api_get_maps)
 def test_concept_map_apply(fhir_concept_map_gender):
-    concept_map = ConceptMap.fetch(fhir_concept_map_gender["id"])
-    concept_map.columns.append("PATIENTS.GENDER")
+    concept_map = ConceptMap(fhir_concept_map_gender["id"])
+
     df = pd.DataFrame({"pk": [1, 2, 3, 4], "PATIENTS.GENDER": ["M", "F", "M", "F"]})
 
-    for col, values in concept_map.apply(df, "pk"):
-        assert col in concept_map.columns
-        assert all([x in ["male", "female"] for x in values])
+    mapped_col = concept_map.apply(df["PATIENTS.GENDER"], "pk")
+
+    assert all(mapped_col == ["male", "female", "male", "female"])
```

Comment:
do you need all() here ?

Reply:
Yes, because "the truth value of an array with more than one element is ambiguous."